src/Search/Search.Management.Tests/project.json

```diff
@@ -1,33 +1,39 @@
 {
-  "version": "1.0.0-*",
+  "version": "1.0.0",
   "description": "Search.Management.Tests Class Library",
   "authors": [ "Microsoft Corporation" ],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
-  "commands": { "test": "xunit.runner.dnx" },
-  "compilationOptions": {
+
+  "packOptions": {
+    "summary": "Generator.CSharp Tests.",
+    "tags": [ "" ],
```

Comment:
Incorrect package summary. This is a pseudo-MAML used only for testing the data plane Search SDK.